packages/ffe-account-selector-react/src/index.d.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, HTMLAttributes, ChangeEventHandler } from 'react';
+import { Account } from './util/types';
+
+interface AccountSelectorProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
This should not inherit from HTMLAttributes<HTMLDivElement> as extra props defined in HTMLAttributes<HTMLDivElement> are not passed into any div element created by AccountSelector.